src/test/TargetSelect/TargetSelect.test.tsx

```diff
@@ -55,14 +55,15 @@ const cryostatAnnotation = {
   REALM: CUSTOM_TARGETS_REALM,
 };
 const mockFooTarget: Target = {
+  jvmId: 'abcd',
   connectUrl: mockFooConnectUrl,
   alias: 'fooTarget',
   annotations: {
     cryostat: cryostatAnnotation,
     platform: {},
   },
 };
-const mockBarTarget: Target = { ...mockFooTarget, connectUrl: mockBarConnectUrl, alias: 'barTarget' };
+const mockBarTarget: Target = { ...mockFooTarget, jvmId: 'efgh', connectUrl: mockBarConnectUrl, alias: 'barTarget' };
```

Comment:
Maybe a test to check if the expanded codeblock is actually shown?

Reply:
Added

Reply:
Actually, just a tiny suggestion here: maybe we should also check things visually alongside with tags so its closer to what users see.